tenseal/tensors/ckksvector.cpp

```diff
@@ -85,7 +85,15 @@ CKKSVector& CKKSVector::add_inplace(CKKSVector to_add) {
     }
 
     if (this->size() != to_add.size()) {
```

Comment:
can to_add have length 0?

Reply:
Actually yeah, thanks for spotting this! We should open an issues about this, as all operations will need this additional check and maybe we will refactor all those check in some way.

Reply:
currently, it's possible to have an empty vector.
check #46

Reply:
Yeah there is already an issue about encrypted vectors, but we should also check plain vectors, which can be passed like `encrypted_vector + []`

Reply:
Although it doesn't work, but maybe a better error message can be used here, and since the checks are repeated many time, might be worth some refactoring